### DISCUSSION TITLE
Serialize protected/private members through they memptr

### DISCUSSION THREAD

**Discussion by schaumb** [stephenberry/glaze#638] [Ideas]
There was a [question](https://github.com/stephenberry/glaze/discussions/294#discussion-5181391), which can be solved easier than requiring getters-setters. The method is simple, use [explicit template instantiation](https://github.com/martong/access_private) because there is [no (member) access check](https://eel.is/c++draft/temp.spec#general-6).

**Comment by stephenberry** [comment]
@schaumb, thanks a ton! That's really neat!
